pkg/onboarding/presentation/config.go

```diff
@@ -71,10 +73,14 @@ func Router(ctx context.Context) (*mux.Router, error) {
 	openSourceUsecases := osusecases.NewUsecasesInteractor(infrastructure, baseExt, pinExt)
 
 	// initialize internal infrastructure
-	infra := infra.NewInteractor()
+	infras := infra.NewInteractor()
```

Comment:
Variable naming ---- _infra_ not _infras_ should make more sense if don't want to use full name